keps/sig-node/4800-cpumanager-split-uncorecache/README.md

```diff
@@ -0,0 +1,676 @@
+
+Yes this feature can be disabled it will just require a restart of `kubelet`. The Kubelet configuration will need to be set with the static policy option and prefer-align-cpus-by-uncorecache flag removed.
+
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+Feature will be enabled. Proper drain of node and restart of kubelet required. Feature is not intended to be enabled/disabled dynamically, similar to static policy.
+
+###### Are there any tests for feature enablement/disablement?
+
+Option is not enabled dynamically. To enable/disable option, cpu_manager_state must be removed and kubelet must be restarted.
+Unit tests will be implemented to test if the feature is enabled/disabled.
+E2e node serial suite can be use to test the enablement/disablement of the feature since it allows the kubelet to be restarted.
+
+
+### Rollout, Upgrade and Rollback Planning
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+Kubelet restarts are not expected to impact existing CPU assignments to already running workloads
+
+
+###### What specific metrics should inform a rollback?
+
+N/A
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+N/A
+
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+N/A
+
+
+### Monitoring Requirements
+
+Include CacheID info in podresources API to be able to verify assignment.
+
+###### How can an operator determine if the feature is in use by workloads?
+
+Reference podresources API to determine CPU assignment and CacheID assignment.
+
+###### How can someone using this feature know that it is working for their instance?
+
+Reference podresources API to determine CPU assignment and CacheID assignment.
+
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+
+- [ ] Events
+  - Event Reason: 
+- [ ] API .status
+  - Condition name: 
+  - Other field: 
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+Measure the time to deploy pods under default settings and compare to the time to deploy pods with align-by-uncorecache enabled. Time difference should be negligible.
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+- Metrics
+  - Pod deployment time (seconds)
```

Comment:
Can you point out specific metric here?

Reply:
We don't specifically have a known metric to measure this, that's part of our proposal to add a new metric would we mention that here?

Reply:
For this and for https://github.com/kubernetes/enhancements/pull/4810#discussion_r1784594941 we can use `topology_manager_admission_duration_ms`  - topology manager since GA will _always_ be engaged even if no NUMA alignment is requested, so this is a good approximation. I'd be open to add more metrics in the future, but this can be a nice start and perhaps enough for alpha.

Reply:
okay I added something